patches/echarts+5.5.0.patch

```diff
@@ -69,3 +70,21 @@ index c25de1b..f94690e 100644
    // Add stack dimension, they can be both calculated by coordinate system in `unionExtent`.
    // That put stack logic in List is for using conveniently in echarts extensions, but it
    // might not be a good way.
+diff --git a/node_modules/echarts/lib/label/labelLayoutHelper.js b/node_modules/echarts/lib/label/labelLayoutHelper.js
+index a75f606..57bec85 100644
+--- a/node_modules/echarts/lib/label/labelLayoutHelper.js
++++ b/node_modules/echarts/lib/label/labelLayoutHelper.js
+@@ -256,6 +256,13 @@ export function hideOverlap(labelList) {
```

Comment:
It would be possible to get this behavior without patching ECharts by using labelLayout and shifting empty labels to one single place outside of the chart but it would be quite hackish. So I agree that ECharts should ignore empty labels when computing layout